frontend/packages/console-dynamic-plugin-sdk/src/shared-modules-override.ts

```diff
@@ -18,6 +18,7 @@ const overrides = {
     require('@console/dynamic-plugin-sdk/src/lib-internal-kubevirt'),
   '@patternfly/react-core': async () => () => require('@patternfly/react-core'),
   '@patternfly/react-table': async () => () => require('@patternfly/react-table'),
+  '@patternfly/react-topology': async () => () => require('@patternfly/react-topology'),
```

Comment:
We want to be cautious about adding things to shared modules because there's no tree shaking or lazy loading. The entire package will be added up front. This package isn't enormous, but I want to be careful what we add to this list. Is this the only option? Do things work if kubevirt references its own version of react-topology?

Reply:
@spadgett `analyze` will fail with vendor being too big without this change.
This seems in place because `react-topology` will likely be used by plugins (kubevirt one of them)
we can always optimize later, i'm sure this wont be the final PR editing `internal-kubevirt`